dev/reference/tools/data-migration/configure/task-configuration-file-full.md

```diff
@@ -0,0 +1,167 @@
+
+  -
+    source-id: "mysql-replica-02"
+    mydumper-config-name: "global"
+    loader-config-name: "global"
+    syncer-config-name: "global"
+```
+
+## Configuration order
+
+1. Edit the [global configuration](#global-configuration).
+2. Edit the [instance configuration](#instance-configuration) based on the global configuration.
+
+## Global configuration
+
+### Basic configuration
+
+Refer to the comments in the [template](#task-configuration-file-template-advanced) to see more details. Detailed explanations about `task-mode` are as follows:
+
+- Description: the task mode that can be used to specify the data replication task to be executed.
+- Value: string (`full`, `incremental`, or `all`).
+    - `full` only makes a full backup of the upstream database and then imports the full data to the downstream database.
+    - `incremental`: Only replicates the incremental data of the upstream database to the downstream database using the binlog. You can set the `meta` configuration item of the instance configuration to specify the starting position of incremental replication.
+    - `all`: `full` + `incremental`. Makes a full backup of the upstream database, imports the full data to the downstream database, and then uses the binlog to make an incremental replication to the downstream database starting from the exported position during the full backup process (binlog position).
+
+### Feature configuration set
+
+Arguments in each feature configuration set are explained in the comments in the [template](#task-configuration-file-template-advanced).
+
+| Parameter        | Description                                    |
+| :------------ | :--------------------------------------- |
+| `routes` | The routing mapping rule set between the upstream and downstream tables. If the names of the upstream and downstream schemas and tables are the same, this item does not need to be configured. See [Table Routing](/dev/reference/tools/data-migration/features/overview.md#table-routing) for usage scenarios and sample configurations. |
+| `filters` | The binlog event filter rule set of the matched table of the upstream database instance. If binlog filtering is not required, this item does not need to be configured. See [Binlog Event Filter](/dev/reference/tools/data-migration/features/overview.md#binlog-event-filter) for usage scenarios and sample configurations. |
+| `black-white-list` | The filter rule set of the black white list of the matched table of the upstream database instance. It is recommended to specify the schemas and tables that need to be replicated through this item, otherwise all schemas and tables will be replicated. See [Binlog Event Filter](/dev/reference/tools/data-migration/features/overview.md#binlog-event-filter)[Black & White Lists](/dev/reference/tools/data-migration/features/overview.md#black--white-table-lists) for usage scenarios and sample configurations. |
```

Comment:
otherwise all schemas and tables will be replicated -> otherwise all schemas and tables **are** replicated